pkg/controllers/user/helm/common.go

```diff
@@ -142,10 +145,17 @@ func (l *Lifecycle) generateTemplates(obj *v3.App) (string, string, *common.Helm
 	if err != nil {
 		return "", "", nil, err
 	}
-
-	commands := append([]string{"template", appDir, "--name", obj.Name, "--namespace", obj.Spec.TargetNamespace}, setValues...)
-
-	cmd := exec.Command(helmName, commands...)
+	if obj.Status.HelmVersion != "helm_v3" {
```

Comment:
can we swap this to be an equals check? And it should use the const `== helmNameV2`